debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/OracleConnectorConfig.java

```diff
@@ -139,6 +145,7 @@
             .withEnum(ConnectorAdapter.class, ConnectorAdapter.LOG_MINER)
             .withWidth(Width.MEDIUM)
             .withImportance(Importance.HIGH)
+            .withGroup(Field.createGroupEntry(Field.Group.CONNECTION, 12))
```

Comment:
I'm not sure whats the difference between `CONNECTION` and `CONNECTOR` from a UI perspective but I don't feel that this option is more attributable to the latter than the former.  Furthermore (at least downstream) we probably should always default this to `logminer` and it be immutable, no?

Reply:
Agree too! I think `CONNECTOR` might really be the better group. Yes, we might exclude this property for specific environments.

Reply:
This is how the backend groups map to the wizard UI pages:

CONNECTION --> Properties - Basic
CONNECTION_ADVANCED  --> Properties - Advanced
CONNECTION_ADVANCED_SSL --> Properties - Advanced / SSL
CONNECTION_ADVANCED_REPLICATION --> Properties - Advanced / Replication
CONNECTION_ADVANCED_PUBLICATION --> Properties - Advanced / Publication
FILTERS --> Filters
CONNECTOR_SNAPSHOT --> Data Options - Snapshot
CONNECTOR --> Data Options - Mapping
CONNECTOR_ADVANCED --> Data Options - Mapping / Advanced
ADVANCED --> Runtime Options - Engine
ADVANCED_HEARTBEAT --> Runtime Options - Heartbeat

As it is currently, I've put all of the logminer properties in CONNECTION_ADVANCED [Properties - Advanced] in the UI